### ISSUE TITLE
您好，我进行测试，但是说没有测试图片，请问一下这个是什么原因

### ISSUE THREAD

**Issue by ZHANG-hengzhi** [Vious/LBAM_Pytorch#10]
D:\anaconda\envs\pytorch\python.exe D:/pycharm/code/LBAM_Pytorch-master/test.py
Input file is not image file!

**Comment by yangwenbing2** [comment]
python test.py --input input_image --mask your_mask --output output_file_prefix --pretrain pretrained_model_path

you should like this, to give you input_image, mask and pretrained_model_path

**Comment by ZHANG-hengzhi** [comment]
thank you very much, i have succeed.



------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2021年6月23日(星期三) 晚上8:40
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [Vious/LBAM_Pytorch] 您好，我进行测试，但是说没有测试图片，请问一下这个是什么原因 (#10)





 
python test.py --input input_image --mask your_mask --output output_file_prefix --pretrain pretrained_model_path
 
you should like this, to give you input_image, mask and pretrained_model_path

**Comment by 01SHENyankun** [comment]
你好，能加个好友嘛，我有些问题想请教，因为是初学者，希望您不要介意，2635505974
